### DISCUSSION TITLE
`interactive` and `active` props for `Card`

### DISCUSSION THREAD

**Discussion by kirillku** [chakra-ui/chakra-ui#8708] [Ideas]
### Description

This feature requests have two separate things, but related to each other. 

- Sometimes the whole card is clickable (`<Card as="a" ... />`) and would be nice to have some styles for hover, would be nice to have `interactive` prop that will enable such styles
- And sometimes you have a list of cards and one of them is active (but still the same variant), would be nice to be able to pass `active` prop and have some styles applied.

### Problem Statement/Justification

I feel it's pretty common use case and it's supported by other ui libs

### Proposed Solution or API

Add two extra props to `Card` component

```tsx
<Card
  interactive // boolean, adds styles for hover
  active // boolean, adds active styles
  ...
/>
```

### Alternatives

We might check if `Card` uses `button` or `a` tag in css and add hover styles 

### Additional Information

I can make a pr with changes, but need some default designs

**Comment by segunadebayo** [comment]
Use `_hover` for a clickable card. Add an `active` variant on the card recipe.

No extra boolean props.
